pkg/cmd/init.go

```diff
@@ -70,6 +70,15 @@ func certSetup(cfg *config.MicroshiftConfig) (*certchains.CertificateChains, err
 		return nil, err
 	}
 
+	externalCertNames := []string{
+		cfg.NodeName,
+		"api." + cfg.BaseDomain,
+	}
+	externalCertNames = append(externalCertNames, cfg.SubjectAltNames...)
+	if cfg.KASAdvertiseAddress != cfg.NodeIP {
```

Comment:
A comment here explaining why we only include the NodeIP if it does not match the kube API server address will help maintainers who are looking at this code later.